doc/hotspot-unit-tests.md

```diff
@@ -172,7 +172,7 @@ Provide informative, but not too verbose error messages.
 All GoogleTest asserts print compared expressions and their values, so
 there is no need to have them in error messages. Asserts print only
 compared values, they do not print any of interim variables, e.g.
-`ASSERT_TRUE((val1 == val2 && isFail(foo(8)) || i == 18)` prints only
+`ASSERT_TRUE(val1 == val2 && isFail(foo(8)) || i == 18)` prints only
```

Comment:
Looking at this expression, I believe the intention is this.
```suggestion
`ASSERT_TRUE((val1 == val2 && isFail(foo(8))) || i == 18)` prints only
```

Reply:
Thanks for the suggestion, updated.